From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

FrancescaLazzeri committed:
Docs article for vienna tutorial deep learning for predictive maintenance

FrancescaLazzeri committed:
adding images for deep learning for predictive maintenance tutorial with Vienna

FrancescaLazzeri committed:
Update deep-learning-for-predictive-maintenance-azure.md

FrancescaLazzeri committed:
Update deep-learning-for-predictive-maintenance-azure.md

mrbullwinkle committed:
Smart Detection 001

FrancescaLazzeri committed:
Moving th efiles to the right folder

FrancescaLazzeri committed:
Change names and add toc file

FrancescaLazzeri committed:
Delete image folder

Commit:
PowerShell

FrancescaLazzeri committed:
Edited .md file

Commit:
formatting

FrancescaLazzeri committed:
Made a few edits

Commit:
CLI

Commit:
acrolinx

Commit:
path

Commit:
formatting

Commit:
resizing art

mrbullwinkle committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into mrb11272017

Commit:
Table

Commit:
get rid of dash

Commit:
fix table

Commit:
heading

Commit:
cough

Commit:
squishing graphic

Commit:
all

kgremban committed:
Review Functions tutorial

kgremban committed:
Consolidate separate code changes into one

kgremban committed:
Finished edits

kgremban committed:
merge conflict

kgremban committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into nov30-functions

Commit:
rticles/site-recovery/site-recovery-vmware-deployment-planner-cost-estimation.md
        articles/site-recovery/site-recovery-vmware-deployment-planner-run.md
        includes/site-recovery-deployment-planner-support-matrix.md

nothing added to commit but untracked files present (use "git add" to track)

v-deasim@SEA-dsimpson MINGW64 ~/azure-docs-pr (master)

chrissie926 committed:
add iot-edge glossary page

itechedit committed:
edit pass: load-balancer-ipv6-for-linux

itechedit committed:
Update load-balancer-ipv6-for-linux.md

itechedit committed:
Update load-balancer-ipv6-for-linux.md

itechedit committed:
Update load-balancer-ipv6-for-linux.md

rwike77 committed:
reforatted

rwike77 committed:
more updates

rwike77 committed:
updated text

rwike77 committed:
minor updates

rwike77 committed:
updated text

rwike77 committed:
updated text

mrbullwinkle committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into mrb11272017

rwike77 committed:
swapped paragraphs

rwike77 committed:
added support

rwike77 committed:
added sample apps

rwike77 committed:
updated samples

rwike77 committed:
reverting

itechedit committed:
Update load-balancer-basic-sku-include.md

chrissie926 committed:
fix review comments

FrancescaLazzeri committed:
made a few corrections

andretms committed:
Merge remote-tracking branch 'upstream/master'

andretms committed:
Merge remote-tracking branch 'upstream/master'

klaaslanghout committed:
Update storage-sync-files-planning.md

customer complained that the doc implies server volume root support (and it's not).  made change to clarify and saying we will support in the near future (internal eta is 1 month away).

MicrosoftGuyJFlo committed:
Changes from public repo

MikeDodaro committed:
New files and image for tutorial about /details endpoint image post

andretms committed:
Moving Guided Setup files to /include. Updating references

andretms committed:
Updating Guided Setup parent files to point to /includes folder

MikeDodaro committed:
Add tutorial to TOC.

andretms committed:
Merge remote-tracking branch 'upstream/master'

andretms committed:
Merge branch 'master' into GuidedSetup-Update-Includes

MikeDodaro committed:
Revise after reveiw

MikeDodaro committed:
Add section about using the ImageInsightsToken.

MikeDodaro committed:
Revise after review 3

mrbullwinkle committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into mrb11272017

MikeDodaro committed:
RecognizedEntities include people and/or places visible in an image

mrbullwinkle committed:
minor fixes

MikeDodaro committed:
Format ImageInsightsToken

daden-ms committed:
update the prerequisites

daden-ms committed:
update links

MikeDodaro committed:
Using my github name as author and microsoft alias as ms.author

mrbullwinkle committed:
sentence structure

MikeDodaro committed:
Found a dead link

mrbullwinkle committed:
minor fix

MikeDodaro committed:
Fix style bugs

mrbullwinkle committed:
TOC update

daden-ms committed:
update data description, update architecture

CawaMS committed:
update SE release not for 0.9.3

CawaMS committed:
correct typo in release note

mrbullwinkle committed:
caps issue

mrbullwinkle committed:
spelling error

MikeDodaro committed:
Fix case

andretms committed:
Fixing Hyperlinks

FrancescaLazzeri committed:
Final review from Francesca

rwike77 committed:
Freshness update

jeffpatt24 committed:
Update storage-sync-files-deployment-guide.md

Updated steps to search for Azure File Sync in the portal

andretms committed:
Merge remote-tracking branch 'upstream/master'

barbkess committed:
add JRJs video

andretms committed:
Pushing changes that were not detected by docs

andretms committed:
Merge branch 'master' into GuidedSetup-Update-Includes

curtand committed:
redirect two classic portal articles

mrbullwinkle committed:
Modify H3 to H2

Commit:
updates for new marketing push

ShawnJackson committed:
Update load-balancer-ipv6-for-linux.md

barbkess committed:
move elastic query

Lisaco88 committed:
Merge pull request #29567 from MikeDodaro/imagePostTutorial

New files and image for tutorial about /details endpoint image post

andretms committed:
Fixing broken hyperlinks

Commit:
adding include to support marketing announcement

jeffpatt24 committed:
Update storage-sync-files-troubleshoot.md

Added server endpoint deletion issue. Removed cloud endpoint deletion issue which is now fixed

v-shils committed:
Merge pull request #29563 from MicrosoftGuyJFlo/AADPublicPRs

Changes from public repo

Commit:
adding fwlink, which will be updated to point to the blog post

Commit:
fixing typo

barbkess committed:
update link

Commit:
Merge pull request #29600 from CawaMS/master

Update Storage Explorer release notes for 0.9.3

ShawnJackson committed:
Merge pull request #29402 from itechedit/load-balancer-ipv6-for-linux

edit pass: load-balancer-ipv6-for-linux

barbkess committed:
fix URL

cwatson-cat committed:
Retiring outdated Login CLI topic.

andretms committed:
Fixing additional broken hyperlinks

mimig1 committed:
Cosmos DB: Updated embedded video for Mongo.

Lisaco88 committed:
Merge pull request #28542 from mrbullwinkle/mrb11272017

Smart Detection 001

mimig1 committed:
Cosmos DB: Embedded connect video.

Lisaco88 committed:
Merge pull request #29401 from chrissie926/master

add iot-edge glossary page

KumudD committed:
removed links to classic portal topic

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into RemoveClassicPortallinks1208

barbkess committed:
move video down

FrancescaLazzeri committed:
Changed text based on Lisa Constantino feedback

PRMerger15 committed:
Merge pull request #29610 from barbkess/videos

add JRJs video

PRMerger17 committed:
Merge pull request #29618 from mimig1/patch-176

Cosmos DB: Embedded connect video.

Commit:
Merge pull request #29616 from mimig1/patch-175

Cosmos DB: Updated embedded video for Mongo.

FrancescaLazzeri committed:
Made a few changes based on Lisa feedback

v-shils committed:
Merge pull request #29599 from daden-ms/daden-patch2

Daden patch2

rwike77 committed:
minor updates

kgremban committed:
Merge conflict

PRMerger15 committed:
Merge pull request #29620 from KumudD/RemoveClassicPortallinks1208

Remove classic portal links

PRMerger16 committed:
Merge pull request #29607 from rwike77/patch-2

Freshness update

Commit:
got link from engineering/marketing

chrissie926 committed:
modify target condition for azure iot hub

v-shils committed:
Merge pull request #29612 from barbkess/toc

move elastic query

Commit:
renaming include to enhancements

garyericson committed:
Changing author from someone no longer at Microsoft

v-thepet committed:
fix broken links

PRMerger17 committed:
Merge pull request #29625 from garyericson/12-08-change-author-to-me

Changing author from someone no longer at Microsoft

Lisaco88 committed:
Merge pull request #28571 from FrancescaLazzeri/master

adding deep learnin scenario

mrbullwinkle committed:
abnormal rise in exceptions article

mrbullwinkle committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into mrb12082017

PRMerger15 committed:
Merge pull request #29626 from v-thepet/newfix

production bug: fix broken links

GitHubber17 committed:
Merge pull request #29009 from kgremban/nov30-functions

Review Functions tutorial

mrbullwinkle committed:
TOC wrap issue

GitHubber17 committed:
Merge pull request #29611 from curtand/classic1208

redirect two classic portal articles

dkamstra committed:
Updated rate limiting content to reflect latest behavior

HJToland3 committed:
mysql limits article update

HJToland3 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into localworking

Commit:
fixing link

Ja-Dunn committed:
Merge pull request #29622 from rwike77/tocupdate

minor updates

Ja-Dunn committed:
Merge pull request #29634 from HJToland3/master

MySQL Limits article update

Ja-Dunn committed:
Merge pull request #29613 from Blackmist/freshness

Freshness

cwatson-cat committed:
Replaced classic portal link with new portal link

Ja-Dunn committed:
Merge pull request #29629 from mrbullwinkle/mrb12082017

abnormal rise in exceptions article

Ja-Dunn committed:
Merge pull request #29633 from dkamstra/master

Updated rate limiting content to reflect latest behavior

Commit:
edits

Commit:
Update analysis-services-overview.md

PRMerger17 committed:
Merge pull request #29637 from cwatson-cat/12-8Cortana-updClassiclink

Replaced classic portal link with new portal link

Ja-Dunn committed:
Merge pull request #28593 from cynthn/short-include

Include

cwatson-cat committed:
Actually deleting the file this time.

PRMerger14 committed:
Merge pull request #29638 from Minewiskan/patch-2

Update analysis-services-overview.md

Commit:
Merge pull request #29624 from chrissie926/master

modify target condition for azure iot hub

garyericson committed:
Fixing 'carolz' as author/manager

Commit:
Revert "       rticles/site-recovery/site-recovery-vmware-deployment-planner-cost-estimation.md"

This reverts commit 10960763856d6fcbc5fbaccb794b5734f31c9144.

undo commits

Ja-Dunn committed:
Merge pull request #29623 from andretms/GuidedSetup-Update-Includes

Change Guided Setups to Include Folders

Commit:
Merge branch 'master' into dean-t591

Ja-Dunn committed:
Merge pull request #29617 from cwatson-cat/12-8-DeleteOldCLILogin

Retiring outdated Login CLI topic

Ja-Dunn committed:
Merge pull request #29643 from dksimpson/dean-t591

Dean t591

Ja-Dunn committed:
Merge pull request #29641 from garyericson/12-08-fix-landing-page-author

Fixing 'carolz' as author/manager

Commit:
Merge pull request #29608 from jeffpatt24/patch-6

Update storage-sync-files-deployment-guide.md

jeffpatt24 committed:
Update storage-sync-files-troubleshoot.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

PRMerger16 committed:
Merge pull request #29559 from klaaslanghout/patch-3

Update storage-sync-files-planning.md

Ja-Dunn committed:
Merge pull request #29416 from rwike77/nextsteps

Updating next steps

PRMerger17 committed:
Merge pull request #29614 from jeffpatt24/patch-7

Update storage-sync-files-troubleshoot.md

v-alje committed:
Merge pull request #29652 from MicrosoftDocs/FromPublicRepo

From public repo
